noirup

```diff
@@ -147,7 +147,7 @@ main() {
     fi
     # Build the repo and install it locally to the .nargo bin directory.
     # --root appends /bin to the directory it is given, so we pass NARGO_HOME.
-    RUSTFLAGS="-C target-cpu=native" ensure cargo install --path ./crates/nargo --bins --locked --force --root $NARGO_HOME
+    RUSTFLAGS="-C target-cpu=native" ensure cargo install --path ./crates/nargo_cli --bins --locked --force --root $NARGO_HOME
```

Comment:
I've removed the usage of `cargo install` to avoid this path dependence. We now just build the nargo bin and pull it out from the `target` directory and move it to where we want it to go.